README.md

```diff
@@ -8,22 +8,77 @@
   <br>
 </h1>
 
-# Features
-- hot reloading
-- basic functionality
+This module implements the [`rdk:component:sensor` API](https://docs.viam.com/components/sensor) and provides two sensor models:
 
-# Models
 ```
 zaporter:sds011:v1
 zaporter:sds011:v1-fake
 ```
-# Example Config
+
+The `zaporter:sds011:v1` model supports the SDS011 Nova PM air quality sensor.
+The `zaporter:sds011:v1-fake` model can be used for testing the module without hardware.
+
+[!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+
+## Features
+
+- Hot reloading
+- Basic functionality
+
+## Configure your SDS011 sensor
+
+> [!NOTE]
+> Before configuring your sensor, you must [create a machine](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine).
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**.
+Select the `sensor` type, then select the `sds011:v1` model.
+Click **Add module**, then enter a name for your sensor and click **Create**.
```

Comment:
I worry about how specific this is because if the UI changes, this will become out of date (and wrong instructions are worse than no instructions imo).

Could this be simplified to something like `[Add this component to your machine](link to docs page with nice pictures adding some other modular component to a machine)`

and then have the attributes info?

Not sure how to balance this. I trust you if you think it is the right thing to do... just worry about fragmentation.

Reply:
As above, TY!